SYCL/ESIMD/lsc/Inputs/lsc_usm_block_load_prefetch.hpp

```diff
@@ -143,6 +143,13 @@ template <typename T> bool test_lsc_block_load() {
     Passed &= test<T, 2, DS, L1H, L3H, NoPrefetch, CheckMerge>(Q, 5, 5);
   if constexpr (sizeof(T) >= sizeof(int))
     Passed &= test<T, 1, DS, L1H, L3H, NoPrefetch, CheckMerge>(Q, 3, 5);
+  // Only 512-bits maximum can be loaded at once (i.e. 4*128 bytes).
```

Comment:
@fineg74 - there was a conflicting change here that I resolved: https://github.com/intel/llvm-test-suite/pull/1590/commits/85f3753ae858be628f6380a6e908e4193443b6f9

But that seems to be not a complete fix. This test needs some additional adjustments such as passing overaligned<8> argument and aligned memory allocation

Reply:
Please add passing overaligned<8> tag for such cases.

Reply:
I couldn't find info in USM spec about the default alignment for malloc_shared().
Memory allocation on the line 42 _probably_ must use aligned_alloc_shared()

Reply:
Ok, thank you. That means that allocation of bytes guarantees only alignment by 1-byte. 
In practice that would be always a bigger alignment, but formally we have to use aligned_alloc_shared<int8_t>(8, ...) to be on safe side.